fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -141,6 +146,10 @@
 
     protected abstract String externalPath();
 
+    protected Deskolemizer deskolemizer() {
+        return deskolemizer == null ? deskolemizer = new Deskolemizer(translator(), null) : deskolemizer;
```

Comment:
If you are objecting to the ternary operator, then that should be encoded in Checkstyle. If, on the other hand, your qualm is genuinely subjective, it's not a valid criticism.